README.md

```diff
@@ -59,4 +59,28 @@ This will generate the test files into the `api-specification` package. Now the
 npm run docs
 ```
 
-inside the `api-specification` package.
\ No newline at end of file
+inside the `api-specification` package.
+
+### Release
+
+To release the packages to npm, install lerna globally and run
```

Comment:
Just out of curiosity: why do you need `lerna` to be globally installed? I expected it would run from the `\ContainerJS` root folder using the locally installed one...